lib/components/ui/file-input.tsx

```diff
@@ -66,7 +73,7 @@ const FileInput = React.forwardRef<HTMLInputElement, FileInputProps>(
             onChange={e => e.target.files?.length && handleInputChange(e.target.files)}
           />
           <Input
-            defaultValue={fileName}
+            defaultValue={fileName || textValue}
```

Comment:
Может назвать defaultValue, чтобы явно понимать для чего оно?
```suggestion
            defaultValue={fileName || defaultValue}
```